concourse/scripts/apply_security_group_bindings_from_vpc_peering_json.rb

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+
+require "json"
+
+if ARGV.empty? || !File.file?(ARGV[0])
+  abort "Usage: #{$PROGRAM_NAME} /path/to/env_vpc_peering_file.json [--dry-run]"
+end
+
+if ARGV[1] == "--dry-run"
+  dry_run = true
+end
+
+config_data = File.read(ARGV[0])
+
+begin
+  config = JSON.parse(config_data)
+rescue JSON::ParserError => e
+  abort "Error parsing JSON file: #{e.message}"
+end
+
+config.each_with_index do |peer, peer_index|
+  peer_name = peer["peer_name"]
+  bindings = peer["bindings"]
+  if peer_name.nil? || peer_name.empty? || bindings.nil? || bindings.empty?
```

Comment:
Don't we want to still support old config files that don't have the bindings option? Maybe we change the message to just a "skipping" instead of error.

Reply:
updated